config/services.yaml

```diff
@@ -29,5 +29,8 @@ services:
         resource: '../src/Controller/'
         tags: ['controller.service_arguments']
 
+    App\Domain\Git:
+        class: App\Adapter\Symfony\Process\Domain\Git
```

Comment:
Do we need add this into configuration? We used already that implementation in command and it is handled by autowiring.